packages/fxa-settings/src/pages/ResetPassword/ResetPasswordRecoveryPhone/en.ftl

```diff
@@ -0,0 +1,30 @@
+## ResetPasswordRecoveryPhone page
+
+reset-password-recovery-phone-flow-heading = Reset your password
```

Comment:
Could use the existing `password-reset-flow-heading` id for this string, since it's shared across reset password pages.